lib/ssh_crypto_policy.pm

```diff
@@ -0,0 +1,109 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+# Summary: Package for ssh cryptographic policy testing
+#
+# Maintainer: George Gkioulis <ggkioulis@suse.com>
+
+package ssh_crypto_policy;
+use base 'consoletest';
+use testapi;
+use utils;
+use strict;
+use warnings;
+
+sub new() {
+    my ($class, %args) = @_;
+    my $self = $class->SUPER::new();
+    $self->{name}            = $args{name};
+    $self->{query}           = $args{query};
+    $self->{cmd_option}      = $args{cmd_option};
+    $self->{algorithm_array} = ();
```

Comment:
Do you need to check if the parameters were passed?

Reply:
This library is used only from `tests/console/sshd.pm` so I think there is currently no need for such check. I can of course add them. @ggkioulis WDYT?

Reply:
Well since this just plays the role of a constructor, I think there is no need for parameter checking, and there is no reason for this package to be called anywhere else.

Reply:
Well since this just plays the role of a constructor, I think there is no need for parameter checking, and there is no reason for this package to be called anywhere else.